flips/20201125-bip-44-multi-account.md

```diff
@@ -0,0 +1,196 @@
+In the long term, it affects Flow wallet developers and their users.
+
+## User Benefit
+
+By following this standard, developers of existing BIP 44 crypto wallets will
+be able to reduce friction for users who wish to create accounts on Flow.
+
+## Design Proposal
+
+### Path Levels
+
+BIP 44 defines an implementation of 
+[Bitcoin Improvement Proposal 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (BIP 32)
+that standardizes a format for hierarchical deterministic (HD) 
+key derivation paths that support multiple coins and multiple accounts per coin.
+
+The BIP 44 path format is as follows:
+
+```
+m / purpose' / coin_type' / account' / change / address_index
+```
+
+When using this format to construct or discover a Flow user wallet, 
+the value(s) of each path parameter should follow the specifications below.
+
+#### Purpose
+
+Purpose is always the constant 44' (0x8000002C), 
+as described in [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#purpose).
+
+#### Coin Type
+
+Coin type is always the constant 539' (0x8000021b), 
+the unique identifier for the Flow protocol, as defined in 
+[SatoshiLabs Improvement Proposal 44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) (SLIP 44).
+
+#### Account
+
+Account is the index of a distinct account address owned by the wallet user. 
+No two account indices should be used to generate child public keys that are assigned 
+to the same Flow account address. Furthermore, a single account index should not
+be used to generate keys for more than one account address.
+
+The motivation for these accounts is the same set forth in BIP 44:
+
+> Users can use these accounts to organize the funds in the same fashion as bank accounts; for donation purposes (where all addresses are considered public), for saving purposes, for common expenses etc.
+
+In addition to the above, Flow wallets should follow the standards for account numbering, 
+creation and discovery described in [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#account).
+
+#### Change
+
+Unlike Bitcoin, Flow does not utilize change addresses. 
+Change is always the constant 0.
+
+#### Index
+
+In BIP 44, each index generates a public key that is used to derive
+a unique address for an account.
+Although Flow does not derive account addresses from public keys,
+index can still be used to generate multiple public keys for a single account.
+
+When applying BIP 44 to Flow, each subsequent value for index should be used to
+generate a new public key for the same account, starting from 0 and incrementing
+by 1.
+
+This index should not be confused with the on-chain account key index.
+
+### Account Discovery
+
+Wallet developers should use a modified version of the original account discovery
+procedure [described in BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#account-discovery).
+
+1. Derive the first account's node (index = 0).
```

Comment:
I feel like the `node` terminology is first introduced here. May be worth more definition before use? Or is it self explanatory?

Reply:
"Node" is a term carried over from BIP 44, but good point -- it isn't explained here. I'll add more clarity to this section